src/deploy-cromwell-on-azure/scripts/mysql/init-user.sql

```diff
@@ -1,5 +1,2 @@
-CREATE USER '{MySqlUserLogin}'@'localhost' IDENTIFIED BY '{MySqlUserPassword}';
-GRANT ALL PRIVILEGES ON {MySqlDatabaseName}.* TO '{MySqlUserLogin}'@'localhost' WITH GRANT OPTION;
-CREATE USER '{MySqlUserLogin}'@'%' IDENTIFIED BY '{MySqlUserPassword}';
-GRANT ALL PRIVILEGES ON {MySqlDatabaseName}.* TO '{MySqlUserLogin}'@'%' WITH GRANT OPTION;
-FLUSH PRIVILEGES;
\ No newline at end of file
+CREATE USER {PostgreSqlUserLogin} WITH PASSWORD '{PostgreSqlUserPassword}';
```

Comment:
Turns out mysql commands don't just work with postgres. I don't know sql very well but this looks to be the equivalent script- please correct me if im wrong. 
Do we still want to support docker mysql even though we'll be deprecating it? If so, I'll have to add two files since the script is so different at this point.

Reply:
The syntax looks correct: https://www.postgresql.org/docs/current/sql-createuser.html